reposerver/src/main/scala/com/advancedtelematic/tuf/reposerver/http/RepoResource.scala

```diff
@@ -226,6 +244,21 @@ class RepoResource(keyserverClient: KeyserverClient, namespaceValidation: Namesp
             }
           }
         }
+      } ~
```

Comment:
The metadata of a target is a sub resource of the Target resource, so it should be under the uri of the target.

targets/<target_id>/comments Where in our case the filename is the target id.

targets/comments/<target_id> implies that comments are a separate resource, but we cannot have comments for a non existent target.

Reply:
Tried that, doesn't work, since TargetFilenamePath eats up the rest of the path.

Reply:
then you could change the path matcher for target filename

Reply:
Was discussing that with Jerry, we preferred to have it under a different resource since it doesn't really belong to TUF data.

Reply:
Changing the path matcher would have meant to check the production DB for filenames with slashes in them.

Reply:
No.